validator/delphi_validator/validate.py

```diff
@@ -0,0 +1,742 @@
+# -*- coding: utf-8 -*-
+"""
+Tools to validate CSV source data, including various check methods.
+"""
+
+import sys
+import re
+import math
+from os.path import join
+from datetime import date, datetime, timedelta
+import pandas as pd
+
+from .errors import ValidationError
+from .datafetcher import filename_regex, \
+    read_filenames, load_csv, get_geo_sig_cmbo, \
+    read_geo_sig_cmbo_files, fetch_api_reference
+
+# Recognized geo types.
+geo_regex_dict = {
+    'county': '^\d{5}$',
+    'hrr': '^\d{1,3}$',
+    'msa': '^\d{5}$',
+    'state': '^[a-z]{2}$',
+    'national': '^[a-z]{2}$'
+}
+
+
+def reldiff_by_min(x, y):
```

Comment:
A style question: It looks like some functions have abbreviations and some not (e.g., reldiff vs relative_difference) is there a delphi coding standard on this? If not would suggest making consistent within the file.

Reply:
No standard except snake_case and appeasing pylint :)